addons/website/migrations/12.0.1.0/pre-migration.py

```diff
@@ -61,17 +61,56 @@ def noupdate_changes(env):
 
 
 def bootstrap_4_migration(env):
-    """Convert customized website views to Bootstrap 4."""
+    """Convert customized website views to Bootstrap 4 and multiwebsite.
+
+    This process is a bit complex, so here's the big picture:
+
+    In v11, there's no upstream support for multi-websites, but databases
+    can actually have several websites and some records (pages, menus, views)
+    get multi-website features behind the scenes, so we have to treat them
+    like semi-supported. Usually it follows this logic:
+
+    - Website-specific records only appear in their website.
+    - Website-agnostic records appear in all websites.
+    - View changes affect all websites.
+
+    In v12, there's full multi-website support, usually following this logic:
+
+    - Website-specific records override website-agnostic ones.
+    - View changes create a website-specific copy (Copy-On-Write, COW) always.
+
+    This migration tries to make v11 changes look like they were done in v12:
+
+    - Unmodified views (noupdate=0) are not touched.
+    - Modified (noupdate=1) views, if they were website-agnostic, are copied,
```

Comment:
Better doc layout and text:
```
- Modified (noupdate=1) views:
  * if they were website-agnostic, they are copied and made 
    website-specific and migrated to Bootstrap 4.
  * if they were website-agnostic, they are marked as noupdate=0
```

Reply:
Both ifs are the same? :thinking:

Reply:
OK, I have read it badly. Then text should be:
```
- Website-agnostic modified (noupdate=1) views are copied and 
  marked as noupdate=0. Copied views are made website-specific
  and migrated to Bootstrap 4.
```